docs/samples/contact-center/app.js

```diff
@@ -153,7 +173,23 @@ function doAgentLogin() {
     console.log('Agent Logged in successfully', response);
   }
   ).catch((error) => {
-    console.log('Agent Login failed', error);
+    console.error('Agent Login failed', error);
+  });
+}
+
+async function handleAgentStatus(event) {
+  const select = document.getElementById('idleCodesDropdown');
+  auxCodeId = event.target.value;
+  agentStatus = select.options[select.selectedIndex].text;
+}
+
+function setAgentStatus() {
+  let state = "Available";
+  if(agentStatus !== 'Available') state = 'Idle';
```

Comment:
According to CC developer portal API there can only be two state either 'Available' or 'Idle' if you pass any other state then the API would fail and will throw Java illegal argument exception.